cowbird/api/webhooks/views.py

```diff
@@ -10,17 +10,20 @@
 from cowbird.services.service_factory import ServiceFactory
 from cowbird.utils import get_logger
 
+logger = get_logger(__name__)
 
-def dispatch(svc_fct):
-    logger = get_logger(__name__)
+
+def dispatch(svc_fct, fct_name):
```

Comment:
I believe the function name could be extracted using `inspect.signature` to avoid this parameter.

Reply:
Tried it, but got "lambda" as the real function is burried inside the lambda :(

Reply:
got it using `inspect.getsource`